### DISCUSSION TITLE
2 Accses IP

### DISCUSSION THREAD

**Discussion by millad221** [Seji64/SniDust#74] [Q&A]
Hi
I want to use an intermediate IP in such a way that the first intermediate IP is 10.10.10.1 (static example), the user sets this in his gateway and requests come to this IP, and on the second server that is 10.10. 10.2 is only able to see the requests from 10.10.10.1 side as a client and give the answer and return what should be done to do this?

**Comment by millad221** [comment]
snidust  | [INFO] Installing default domains...
snidust  | '/var/lib/snidust/domains.d/00-debug.lst' -> '/etc/snidust/domains.d/00-debug.lst'
snidust  | '/var/lib/snidust/domains.d/01-cdn_akamai.lst' -> '/etc/snidust/domains.d/01-cdn_akamai.lst'
snidust  | '/var/lib/snidust/domains.d/02-amazon.lst' -> '/etc/snidust/domains.d/02-amazon.lst'
snidust  | '/var/lib/snidust/domains.d/03-hbo.lst' -> '/etc/snidust/domains.d/03-hbo.lst'
snidust  | '/var/lib/snidust/domains.d/04-hulu.lst' -> '/etc/snidust/domains.d/04-hulu.lst'
snidust  | '/var/lib/snidust/domains.d/05-netflix.lst' -> '/etc/snidust/domains.d/05-netflix.lst'
snidust  | '/var/lib/snidust/domains.d/06-molotov_tv.lst' -> '/etc/snidust/domains.d/06-molotov_tv.lst'
snidust  | '/var/lib/snidust/domains.d/07-srf_ch.lst' -> '/etc/snidust/domains.d/07-srf_ch.lst'
snidust  | '/var/lib/snidust/domains.d/09-zattoo.lst' -> '/etc/snidust/domains.d/09-zattoo.lst'
snidust  | '/var/lib/snidust/domains.d/10-yallo.lst' -> '/etc/snidust/domains.d/10-yallo.lst'
snidust  | '/var/lib/snidust/domains.d/11-youtube.lst' -> '/etc/snidust/domains.d/11-youtube.lst'
snidust  | [INFO] Generating ACL...
snidust  | [ERROR] ALLOWED_CLIENTS_FILE is set but file does not exists or is not accessible!
snidust  | [INFO] Generating DNSDist Config...
snidust  | [INFO] Starting DNSDist...
snidust  | [INFO] Starting nginx..
snidust  | nginx: [emerg] invalid number of arguments in "allow" directive in /etc/nginx/allowedClients.conf:2


I want to give my list, but it gives me an error. How should I enter the access list of IPs and sites?

**Comment by Seji64** [comment]
Sorry i dont't get what your question is or what want to achive

**Comment by millad221** [comment]
See, I have an IP from the Netherlands with 10.10.10.123 (you assume this is the static IP of my virtual server)
I have another server in Elman with 20.20.20.112 (you assume this is the static IP of my virtual server)
Now I want all my users to connect to my Dutch IP, that is, put the IP as DNS in their system, but the traffic will be transferred from the Netherlands to Germany and then back to the Netherlands and respond to the user. Is there a way to do this?

**Reply by Seji64** [reply]
Still don't get the reason to do this, but this is nothing to solve with snidust. what you need is routing. look at iptables and so on. not an easy topic...

**Comment by millad221** [comment]
hello
Is DoH or DoT currently active on the service or should it be done manually?

**Reply by Seji64** [reply]
Currently not implemented

**Comment by millad221** [comment]
Do you need to activate it or can I activate it myself?

**Comment by millad221** [comment]
Can you help me?
Some services that are on TLS cannot be opened and a certificate must be created for them in your image.

**Reply by Seji64** [reply]
Of course you can activate it yourself, but you need a certificate which you have to generate yourself or request from Let's Encrypt, for example. Furthermore you have to adjust the dnsdist config and map the ports (docker).

**Reply by millad221** [reply]
> Of course you can activate it yourself, but you need a certificate which you have to generate yourself or request from Let's Encrypt, for example. Furthermore you have to adjust the dnsdist config and map the ports (docker).

Is it possible to give an example to change?
